src/Whoops/Util/TemplateHelper.php

```diff
@@ -183,7 +183,7 @@ public function dump($value)
             return $output;
         }
 
-        return print_r($value, true);
+        return var_dump($value, true);
     }
```

Comment:
var-dump has no 2nd parameter which makes it return the value as a string.

Also I dont think that var-dump uses different memory then any other funtction

Reply:
Just wondering but isn't `var_export` fails in a recursive array?

Reply:
Not that I know of and I didn't find anything of this liking. The official PHP documentation mentions nothing of the liking. Some user comments refer to this, but those remarks seem to relate to older PHP versions. And that's precisely the issue here: `print_r` is known to misbehave. `var_export` and `var_dump` behave slightly better ...